### ISSUE TITLE
Error with summarize interface

### ISSUE THREAD

**Issue by jpbalarini** [cohere-ai/cohere-typescript#105]
Hello! I'm trying to do a summary using the `summarize` method, but I get a type error. It looks like the interface is not ok.

I'm calling the API like this:
```
const summaryResponse = await cohere.summarize({
  text: documentData,
});

console.log("Summary: ", summaryResponse);
```

And I get something like this:
```
Summary:  {
  id: 'a78f3713-...',
  summary: 'This is a Request for Proposal (RFP) issued ...',
  meta: {
    apiVersion: { version: '1' },
    billed_units: { input_tokens: 3689, output_tokens: 100 }
  }
}
```

But when I actually want to access the summary like this:
```
console.log(summary.summary);
```
I get a types error. Looking at the interface, it looks like it's not correct, as it is expecting a results array (which of course, does not exist):
https://github.com/cohere-ai/cohere-typescript/blob/89c12f0ae5b64fa1577f872bfc150fef507e68ad/src/api/types/SummarizeResponse.ts#L7

**Comment by billytrend-cohere** [comment]
Hey, @jpbalarini thanks for reporting this! This is an issue with our spec so I'm expediting a fix for you. Hope to release today!

**Comment by billytrend-cohere** [comment]
Fixed in `cohere-ai@7.4.1`!

**Comment by jpbalarini** [comment]
That was fast! thank you 🥇
